libs/components/packages/src/schematics/migrations/update-11/vscode-eslint-setting/vscode-eslint-setting.schematic.ts

```diff
@@ -0,0 +1,49 @@
+import { Rule, Tree } from '@angular-devkit/schematics';
+
+import { readRequiredFile, writeJsonFile } from '../../../utility/tree';
+
+function getVSCodeSettings(tree: Tree): {
+  'eslint.useFlatConfig'?: boolean;
+} {
+  if (tree.exists('.vscode/settings.json')) {
```

Comment:
Might be worth saving this file path in a constant since it's used three times in this script.

Reply:
Done. I pulled the package.json path out as well.